src/components/organisms/ImageManager.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {faLink, faUpload} from "@fortawesome/free-solid-svg-icons";
+import Image from "../atoms/Image";
+
+const Images = styled("ul")`
+	padding: 0;
+	list-style: none;
+`;
+
+const ImageRow = styled("li")`
+	display: grid;
+	grid-template-columns: 50px auto 15px;
+	grid-column-gap: 5px;
+	margin-bottom: 15px;
+	line-height: 50px;
+	
+	img {
+		width: 50px;
+		height: 50px;
+	}
+`;
+
+const CopyLink = styled("span")`
+	text-align: right;
+`;
+
+class ImageManager extends Component {
```

Comment:
We can remove this for now until we get image functionality implemented properly.